src/window.rs

```diff
@@ -253,6 +256,22 @@ impl Window {
         };
     }
 
+    /// Sets how the cursor should be handled.
+    ///
+    /// See the documentation for [`CursorState`] for the possible cursor states.
+    ///
+    /// Note that if you use [`CursorState::Grab`], you should use [`Input::mouse_delta_raw`] for
+    /// detecting mouse movement, as [`Input::mouse_delta`] will only report movement of the cursor
+    /// within the window.
+    ///
+    /// [`CursorState`]: enum.CursorState.html
+    /// [`CursorState::Grab`]: enum.CursorState.html#variant.Grab
+    /// [`Input::mouse_delta_raw`]: struct.Input.html#method.mouse_delta_raw
+    /// [`Input::mouse_delta`]: struct.Input.html#method.mouse_delta_raw
+    pub fn set_cursor_state(&self, state: CursorState) {
+        let _ = self.window.set_cursor_state(glutin::CursorState::Grab);
```

Comment:
looks like this code ignores the `state` parameter

Reply:
Womp womp, I totally messed that up. I've pushed a fix.